projects/1.16/assets/fantasys-furniture/fantasyfurniture/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+{
+  "block.fantasyfurniture.decorations.berry_basket_blueberry": "蓝梅篮",
+  "block.fantasyfurniture.decorations.berry_basket_empty": "浆果篮",
+  "block.fantasyfurniture.decorations.berry_basket_strawberry": "草莓篮",
+  "block.fantasyfurniture.decorations.berry_basket_sweetberry": "甜莓篮",
+  "block.fantasyfurniture.decorations.bolts_of_cloth": "布匹",
+  "block.fantasyfurniture.decorations.book_stack": "书架",
+  "block.fantasyfurniture.decorations.book_stack.stacked": "堆积的书堆",
+  "block.fantasyfurniture.decorations.bowl_beetroot_soup": "",
+  "block.fantasyfurniture.decorations.bowl_empty": "碗",
+  "block.fantasyfurniture.decorations.bowl_mushroom_stew": "甜菜根汤碗",
+  "block.fantasyfurniture.decorations.coin_stack_gold": "金币堆",
+  "block.fantasyfurniture.decorations.coin_stack_iron": "铁币堆",
+  "block.fantasyfurniture.decorations.dunmer.pottery_0": "丹莫陶器 1",
+  "block.fantasyfurniture.decorations.dunmer.pottery_1": "丹莫陶器 2",
+  "block.fantasyfurniture.decorations.muffins_blueberry": "蓝莓松饼",
+  "block.fantasyfurniture.decorations.muffins_blueberry.stacked": "堆积的蓝莓",
```

```suggestion
  "block.fantasyfurniture.decorations.muffins_blueberry.stacked": "可堆积",
```
这里的Stackable应当是说明文本之一？
可以先放一张游戏内截图，如果是请全部修改为`可堆积`